src/timeMachine/components/FunctionModeSelector.tsx

```diff
@@ -34,20 +41,35 @@ type ReduxProps = ConnectedProps<typeof connector>
 type Props = ReduxProps
 
 const FunctionSelector: FunctionComponent<Props> = ({
-  onSetFunctions,
-  selectedFunctions,
-  onSingleSelectBuilderFunction,
+  activeGraphType,
+  isInCheckOverlay,
   onMultiSelectBuilderFunction,
+  onSetFunctions,
   onSetIsAutoFunction,
-  isInCheckOverlay,
+  onSingleSelectBuilderFunction,
+  savedFunctions,
 }) => {
-  const autoFunctions = AUTO_FUNCTIONS.map(f => f.name)
+  const autoFunctions = getDefaultAutoFunctions(activeGraphType).map(
+    f => f.name
```

Comment:
i know you you didn't write this, but you would mind maybe just updating `f` to something easier to search for, like `autoFunction`, please?

Reply:
Done.